jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannelState.java

```diff
@@ -1378,6 +1379,7 @@ public boolean onReadPossible()
                 case IDLE:
                 case READY:
                 case REGISTER:
+                case POSSIBLE:
```

Comment:
Again can you explain this?   If we are in POSSIBLE state then a thread is on its way to call handle an switch us to PRODUCING.   So we've already been called back! How can we get called back again?

Reply:
This should not happen, let me fix this.